build/build.sh

```diff
@@ -54,8 +60,8 @@ pandoc --verbose \
 DOCKER_EXISTS="$(command -v docker || true)"
 
 # Create PDF output (unless BUILD_PDF environment variable equals "false")
-if [ "$BUILD_PDF" != "false" ] && [ -z "$DOCKER_EXISTS" ]; then
-  echo "Exporting PDF manuscript using WeasyPrint"
+if [ "${BUILD_PDF:-true}" != "false" ] && [ -z "$DOCKER_EXISTS" ]; then
```

Comment:
Change to use michaelmhoffman's tip 11? https://github.com/manubot/rootstock/pull/233#issuecomment-500898800

Reply:
> You don't need `["${BUILD_PDF:-true}" != "false"]`, just `["${BUILD_PDF:-}" != "false"]`. Better yet, the usual convention is to test `[ "${BUILD_PDF:-}" ]` which will let a user set `BUILD_PDF` to anything to mean true and empty or unset to mean false. Using a command-line argument is a more expected interface in most cases though.

I don't think `[ "${BUILD_PDF:-}" ]` is what we wan't here because it defaults to not building the PDF if `BUILD_PDF` is unset. However, we want PDF building unless explicitly disabled with `BUILD_PDF=false`. I also think we want a solution where `BUILD_PDF=false` and `BUILD_PDF=true` perform as written, such that they can be easily toggled in CI configs. We could do `["${BUILD_PDF:-}" != "false"]`, which saves a few characters but is a bit less readable for the non-expert.

Reply:
You could call it `SUPPRESS_PDF` or better yet `MANUBOT_SUPPRESS_PDF` (or something semantically equivalent).

Another alternative is simply to set the variable upstream with something like:

```
BUILD_PDF="${BUILD_PDF:-true}"
```

Then you can just use `"${BUILD_PDF}"` from then on forward.